data/lua/wml/random_placement.lua

```diff
@@ -14,7 +14,7 @@ wesnoth.wml_actions.random_placement = function(cfg)
 	local math_abs = math.abs	
 	local locs = wesnoth.get_locations(filter)
 	if type(num_items) == "string" then		
-		num_items = math.floor(loadstring("local size = " .. #locs .. "; return " .. num_items)())
+		num_items = math.floor(load("local size = " .. #locs .. "; return " .. num_items)())
```

Comment:
peopel can already execute every lua code using [lua] tags, it really doesnt make sense to think about security issue here.